### DISCUSSION TITLE
What are the typical IP address ranges for Data Landing Zone

### DISCUSSION THREAD

**Discussion by renepajta** [Azure/data-landing-zone#218] [Q&A] [answered]
What are the typical IP address ranges for Data Landing Zone? In the template default is /16 (4096 IP addresses) which looks too much for some organisation. 

Why do we consider such large address range and what are the main services which consumes the IPs? What is the typical size you see at customers?

**Comment by marvinbuss** [comment, marked as answer]
With the default setup, we are just consuming 23 IP addresses. However for the Data landing Zone it is a bit more tricky, because of the following factors:
1. Databricks clusters will consume additional IPs. Hence, we have to size the subnets of the Databricks workspaces properly.
2. Additional Data Integrations and Products are expected over time, whereas each will consume some IP addresses on a separate subnet.

The subnet size for Databricks workspaces cannot be modified after the deployment. As a result, I would look at existing workloads to understand what size the subnets should have. Alternatively, if this is a green field deployment and the customer is rather small, I would use a /24 to start with. This should provide a sufficient address space.

Each data product/integration will also require additional address space. Therefore, I would add some IPs to the allocated address space to be able to deploy use cases over the upcoming months and years.
I hope this helps!
